README.md

```diff
@@ -11,11 +11,85 @@ Main characteristics:
 * can handle both immutable and mutable data;
 * results in compact code with the absolute minimum of boilerplate.
 
-tansu works well with the Angular ecosystem:
+Implementation wise, it is a tiny (500 LOC) library without any external dependencies.
+
+## Comparison with the Svelte stores
+
+Tansu is designed to be and to remain fully compatible with Svelte. Nevertheless, it brings several improvements:
+
+### tansu works well with the Angular ecosystem:
 * works with the standard `async` pipe out of the box;
 * stores can be registered in the DI container at any level (module or component injector).
 
-Implementation wise, it is a tiny (500 LOC) library without any external dependencies.
+### a batch function is available
+
+Depending on multiple stores can lead to some issues. Let's have a look at the following example:
+
+
+The diamond dependency issue is more specific when a store depends on the same ancestor through multiple paths:
+```mermaid
+graph TD;
+    A-->B;
+    A-->C;
+    B-->D;
+    C-->D;
+```
+
+In your example, you do not really have a diamond dependency issue, only a dependency on multiple stores that you want to change in an atomic way.
+
+```mermaid
+graph TD;
+    firstName-->fullName;
+    lastName-->fullName;
+```
```

Comment:
I don't think you should include this in the README.md, it was only intended to be a comment on the PR, not to be included in the file.
I suppose the mistake was done by GitHub when approving my suggestion of change.

It would make sense to add some graphs in the README.md in another PR to explain those issues that the svelte store has and that are solved by tansu (such as the [asymmetric diamond dependency issue](https://github.com/AmadeusITGroup/tansu/pull/31)).

Reply:
Well, in fact, the confusion came from me !

I thought you wanted to include the graph to explain the kind of issues it resolves. But right, we can do it in another PR.